src/main/java/com/dynatrace/loadrunner/converter/util/BodyFilePatcherUtil.java

```diff
@@ -49,7 +49,7 @@ public static String concatTransactionNames(List<String> transactionNames) {
 		return builder.toString();
 	}
 
-	public static int getIndexAfterLineComment(String commentedInstruction, char aktCh, int i) {
+	private static int getIndexAfterLineComment(String commentedInstruction, char aktCh, int i) {
```

Comment:
rename all occurrences of `aktCh` or `aktChar`, get rid of polish prefixes
use e.g.`currentChar`